web/src/components/JsonLdWebSite.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react'
+import { Helmet } from 'react-helmet'
+import { WebSite, WithContext } from 'schema-dts'
+
+import buildConfig from '../constants/buildConfig'
+
+const JsonLdBreadcrumbs = (): ReactElement => {
+  const config = buildConfig()
+
+  const jsonLd: WithContext<WebSite> = {
+    '@context': 'https://schema.org',
+    '@type': 'WebSite',
+    name: config.hostName,
```

Comment:
Shouldn't the name rather be `config.appName`?

Reply:
See ticket description.

Reply:
Missed that, thanks. Where does `Local & Multilingual Information |` come from?

Reply:
I guess that's supposed to be the title of a site. But not sure of which site. I guess we could use this as title for the landing page as "Regionen" does not supply much information